cmd/lakectl/cmd/branch_tools_test.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/treeverse/lakefs/pkg/metastore"
+	"github.com/treeverse/lakefs/pkg/metastore/mock"
+)
+
+const (
+	dbName     = "source"
+	repoName   = "repo"
+	branchName = "branch"
+)
+
+var (
+	validPath   = fmt.Sprintf("s3://%s/%s/path/to/schema", repoName, branchName)
+	invalidPath = fmt.Sprintf("s3/%s/%s/", repoName, branchName)
+)
+
+func TestExtractRepoAndBranchFromDBName(t *testing.T) {
+	type args struct {
+		ctx    context.Context
+		dbName string
+	}
+	tests := []struct {
+		name      string
+		args      args
+		want      string
+		want1     string
+		wantErr   bool
+		validPath bool
+	}{
+		{
+			name: "Happy flow",
```

Comment:
Nit:
```suggestion
			name: "Valid",
```
(or similar).  The other test flows are less happy, but I don't want to say "Sad" about each of them.

Reply:
The other flows are sad from my POV but sure 😉